### PR TITLE
loggin skipped of toiletuser page

### PR DESCRIPTION
**Pull request by thant-sin-aung**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ function App() {
           <ProtectedRoute exact path="/dashboard" component={DashBoard} />
           <ProtectedRoute path="/profiles" component={ProfileSelectionPage} />
           <ProtectedRoute path="/welcome" component={WelcomePage} />
-          <ProtectedRoute path="/toiletUsers" component={ToiletUsersPage} />
+          <Route path="/toiletUsers" component={ToiletUsersPage} />
```
Comment (phyu007) [inline src/App.js:25]: Any reason why ProtectedRoute is removed?

File: src/components/ToiletUsers/toiletUsers.jsx [hunk 2]
```diff
@@ -38,39 +40,76 @@ useEffect(() => {
...
+        });
+      } 
   };
 
   return (
     
     <div className="profile-selection-page"
       style={{ display: 'flex', flexWrap: 'wrap', alignItems: 'center', justifyContent: 'center', flexDirection: 'column', height: '100vh' }}
     >
-      <h1>Select your profile</h1>
```
Comment (phyu007) [inline src/components/ToiletUsers/toiletUsers.jsx:70]: why remove?

File: src/App.js (hunk 1 above)
Comment (thant-sin-aung) [reply src/App.js:25]: to skip login again

File: src/components/ToiletUsers/toiletUsers.jsx (hunk 2 above)
Comment (thant-sin-aung) [reply src/components/ToiletUsers/toiletUsers.jsx:70]: to fit raspberry pi screen

**Pull request merged**